WORKSPACE

```diff
@@ -50,36 +50,9 @@ load("@io_bazel_rules_go//go:deps.bzl", "go_download_sdk", "go_register_toolchai
 
 go_rules_dependencies()
 
```

Comment:
Yeah, we want to make sure to be able to download both the linux and mac sdks regardless of host (the default rules only download the SDK from your host machine). This allows you to kick of linux remote execution builds from a mac host, i.e. https://www.buildbuddy.dev/debugging#run-buildbuddy-test-suite-on-linux-remote-ex-from-a-mac

Reply:
I pushed a fix in https://github.com/bazelbuild/rules_go/pull/3511 and patched our rules_go with it in the latest version.

Since the rules_go PR is merged, it should be a short-lived patch.